js/tarteaucitron/README.md

```diff
@@ -73,14 +71,19 @@ tarteaucitron.init({
     // "customCloserId": "", /* Optional a11y: Custom element ID used to open the panel */
 
     "googleConsentMode": true, /* Enable Google Consent Mode v2 for Google ads and GA4 */
+    "bingConsentMode": true, /* Enable Bing Consent Mode for Clarity and Bing Ads */
+    "softConsentMode": false, /* Soft consent mode (consent is required to load the services) */
+
+    "dataLayer": false, /* Send an event to dataLayer with the services status */
+    "serverSide": false, /* Server side only, tags are not loaded client side */
     
     "partnersList": false /* Details the number of partners on the popup and middle banner */
 });
 </script>
 ```
 
 # Add a service
-[Visit tarteaucitron.io](https://tarteaucitron.io/en/install/)
+[Installation guide](https://tarteaucitron.io/en/free-installation-open-source/)
```

Comment:
I would recommend renaming the link text to something like `Service installation guide` and start with something like "skip to step 3" otherwise people might think it's related to installing the module, and accidentally end up adding the same code twice (if they do steps 1 and 2)